service/matching/matchingEngine.go

```diff
@@ -896,11 +896,6 @@ func (e *matchingEngineImpl) getTask(
 	return tlMgr.GetTask(ctx, maxDispatchPerSecond)
 }
 
-//
-//func (e *matchingEngineImpl) getHistoryInfo() (HistoryInfo, error) {
-//
-//}
-
```

Comment:
huh.  wonder why this (and HistoryInfo's definition) were introduced as commented-out.

Reply:
zero clue